cli/cmd/user/list.go

```diff
@@ -78,7 +84,7 @@ func ListCmd(ch *cmdutil.Helper) *cobra.Command {
 	listCmd.Flags().StringVar(&ch.Org, "org", ch.Org, "Organization")
 	listCmd.Flags().StringVar(&projectName, "project", "", "Project")
 	listCmd.Flags().StringVar(&groupName, "group", "", "User group")
-	listCmd.Flags().Uint32Var(&pageSize, "page-size", 50, "Number of users to return per page")
+	listCmd.Flags().Uint32Var(&pageSize, "page-size", DefaultPageSize, "Number of users to return per page")
```

Comment:
Using a constant here is IMO not helpful and just adds indirection – it is natural to specify the default value for a flag on the line of code that defines the flag